### ISSUE TITLE
fix: 使用REST API 查询用户报错

### ISSUE THREAD

**Issue by DemingCheng** [MemTensor/MemOS#380]
### Pre-submission checklist | 提交前检查

- [x] I have searched existing issues and this hasn't been mentioned before | 我已搜索现有问题，确认此问题尚未被提及
- [x] I have read the project documentation and confirmed this issue doesn't already exist | 我已阅读项目文档并确认此问题尚未存在
- [x] This issue is specific to MemOS and not a general software issue | 该问题是针对 MemOS 的，而不是一般软件问题

### Bug Description | 问题描述

在本地搭建vllm服务后，测试相关路由：创建user成功，sqlite可看到相关用户

<img width="2466" height="951" alt="Image" src="https://github.com/user-attachments/assets/aa664f44-43d9-44bd-8492-f1cd0e45e237" />

<img width="1214" height="298" alt="Image" src="https://github.com/user-attachments/assets/8db16c23-380c-4f4c-beca-d3ceb6b80eb4" />

在查询用户时报错

<img width="1005" height="140" alt="Image" src="https://github.com/user-attachments/assets/53e718df-65a6-46dc-987e-91b459bf4f5f" />

<img width="1790" height="642" alt="Image" src="https://github.com/user-attachments/assets/95047584-edfa-4399-9b6b-80564f30231a" />

### How to Reproduce | 如何重现

1. 启动 VLLM 服务
2. 启动docker 容器：docker compose up
3. 启动MemOS 服务：uvicorn memos.api.product_api:app --host 0.0.0.0 --port 8081 --reload
4. 打开网址：http://10.90.24.114:8081/docs

测试 /product/users 报错

### Environment | 环境信息

Python 3.10.12
ubuntu 22.04

br104@ylf-p-sw-br104oam-server13 /ceph/teamFPA/Distributed/dm/nixl $ pip list
Package                                  Version           Editable project location
---------------------------------------- ----------------- -------------------------------------------------
absl-py                                  2.3.1
accelerator                              2024.9.13
aiofiles                                 25.1.0
aiohappyeyeballs                         2.6.1
aiohttp                                  3.12.15
aiosignal                                1.4.0
airportsdata                             20250909
annotated-types                          0.7.0
anyio                                    4.9.0
argcomplete                              3.6.2
astor                                    0.8.1
astroid                                  3.3.11
asttokens                                3.0.0
async-timeout                            5.0.1
attrs                                    25.3.0
auditwheel                               6.4.2
Authlib                                  1.6.0
Automat                                  20.2.0
Babel                                    2.8.0
bcrypt                                   3.2.0
beautifulsoup4                           4.13.4
biren-ml-py                              1.9.0+br1xx
black                                    25.1.0
blake3                                   1.0.7
blinker                                  1.9.0
blobfile                                 3.0.0
boto3                                    1.40.21
botocore                                 1.40.21
bottle                                   0.12.25
bpex                                     0.5.0+br1xx
br-btas                                  0.3.0+br1xx
br-precision-debugger                    1.8.0+br1xx
cachetools                               6.2.1
ceph                                     1.0.0
cephfs                                   2.0.0
certifi                                  2022.12.7
cffi                                     1.17.1
cfgv                                     3.4.0
chardet                                  4.0.0
charset-normalizer                       2.1.1
chonkie                                  1.1.1
click                                    8.2.1
cloud-init                               23.3.3
cloudpickle                              3.1.1
cmake                                    4.1.0
cobble                                   0.1.4
colorama                                 0.4.6
coloredlogs                              15.0.1
command-not-found                        0.3
compressed-tensors                       0.9.4
configobj                                5.0.6
constantly                               15.1.0
contourpy                                1.3.2
coverage                                 7.10.6
cpplint                                  2.0.2
cryptography                             45.0.5
cupy-cuda12x                             13.6.0
cycler                                   0.12.1
cyclopts                                 3.22.2
datasets                                 4.0.0
dbus-python                              1.2.18
decorator                                4.4.2
deepspeed-br                             2.5.0+br1xx
defusedxml                               0.7.1
Deprecated                               1.2.18
depyf                                    0.18.0
dill                                     0.3.8
diskcache                                5.6.3
distlib                                  0.4.0
distro                                   1.9.0
distro-info                              1.1+ubuntu0.2
dnspython                                2.7.0
docstring_parser                         0.16
docutils                                 0.21.2
einops                                   0.8.1
elastic-toolkit                          0.6.0+br1xx
email_validator                          2.2.0
et_xmlfile                               2.0.0
exceptiongroup                           1.3.0
executing                                2.2.1
expecttest                               0.1.6
fastapi                                  0.115.6
fastapi-cli                              0.0.8
fastapi-cloud-cli                        0.1.4
fastcore                                 1.8.8
fastmcp                                  2.10.5
fastrlock                                0.8.3
filelock                                 3.13.1
flake8                                   4.0.1
Flask                                    3.1.2
flatbuffers                              25.2.10
fonttools                                4.59.2
frozenlist                               1.7.0
fsspec                                   2024.6.1
ftfy                                     6.3.1
gguf                                     0.17.1
gitdb                                    4.0.12
GitPython                                3.1.45
googleapis-common-protos                 1.70.0
greenlet                                 3.2.3
grpcio                                   1.73.1
h11                                      0.16.0
h2                                       4.2.0
h5py                                     3.14.0
hf-xet                                   1.1.5
hjson                                    3.1.0
hpack                                    4.1.0
httpcore                                 1.0.9
httplib2                                 0.20.2
httptools                                0.6.4
httpx                                    0.28.1
httpx-sse                                0.4.1
huggingface-hub                          0.33.4
humanfriendly                            10.0
Hypercorn                                0.17.3
hyperframe                               6.1.0
hyperlink                                21.0.0
identify                                 2.6.13
idna                                     3.4
importlib_metadata                       8.0.0
incremental                              21.3.0
iniconfig                                2.1.0
interegular                              0.3.3
ipython                                  8.37.0
isort                                    6.0.1
itsdangerous                             2.2.0
jedi                                     0.19.2
jeepney                                  0.7.1
Jinja2                                   3.1.4
jiter                                    0.10.0
jmespath                                 1.0.1
joblib                                   1.5.1
jsonlines                                4.0.0
jsonpatch                                1.32
jsonpath                                 0.82.2
jsonpointer                              2.0
jsonschema                               4.24.1
jsonschema-specifications                2025.4.1
keras                                    3.11.3
keyring                                  23.5.0
kiwisolver                               1.4.9
lark                                     1.2.2
launchpadlib                             1.10.16
lazr.restfulclient                       0.14.4
lazr.uri                                 1.0.6
libsutx                                  1.2.2
llguidance                               0.7.30
llvmlite                                 0.44.0
lm-dataformat                            0.0.20
lm-format-enforcer                       0.10.12
lxml                                     6.0.0
magika                                   0.6.2
mammoth                                  1.9.1
Markdown                                 3.8.2
markdown-it-py                           3.0.0
markdownify                              1.1.0
markitdown                               0.1.2
MarkupSafe                               3.0.2
matplotlib                               3.10.6
matplotlib-inline                        0.1.7
mccabe                                   0.6.1
mcp                                      1.12.0
mdurl                                    0.1.2
megatron-br                              0.10.0.2507+br1xx
megfile                                  4.2.4
MemoryOS                                 1.1.2
meson                                    1.8.3
mistral_common                           1.8.5
ml_dtypes                                0.5.3
modelscope                               1.30.0
more-itertools                           8.10.0
mpmath                                   1.3.0
msgpack                                  1.1.2
msgspec                                  0.19.0
multidict                                6.6.4
multiprocess                             0.70.16
mypy_extensions                          1.1.0
namex                                    0.1.0
neo4j                                    5.28.1
nest-asyncio                             1.6.0
netifaces                                0.11.0
networkx                                 3.3
ninja                                    1.13.0
nodeenv                                  1.9.1
numba                                    0.61.2
numpy                                    1.26.0
oauthlib                                 3.2.0
ollama                                   0.4.9
onnxruntime                              1.23.1
openai                                   1.97.0
openapi-pydantic                         0.5.1
opencv-python-headless                   4.11.0.86
openpyxl                                 3.1.5
opentelemetry-api                        1.26.0
opentelemetry-exporter-otlp              1.26.0
opentelemetry-exporter-otlp-proto-common 1.26.0
opentelemetry-exporter-otlp-proto-grpc   1.26.0
opentelemetry-exporter-otlp-proto-http   1.26.0
opentelemetry-proto                      1.26.0
opentelemetry-sdk                        1.26.0
opentelemetry-semantic-conventions       0.47b0
opentelemetry-semantic-conventions-ai    0.4.13
optree                                   0.17.0
orjson                                   3.11.0
outlines                                 0.1.11
outlines_core                            0.1.26
packaging                                25.0
pandas                                   2.3.1
paramiko                                 3.5.1
parso                                    0.8.5
partial-json-parser                      0.2.1.1.post6
patchelf                                 0.17.2.4
pathspec                                 0.12.1
pdfminer.six                             20250506
pexpect                                  4.8.0
pika                                     1.3.2
pillow                                   11.0.0
pip                                      25.2
platformdirs                             4.4.0
pluggy                                   1.6.0
portalocker                              2.10.1
pre_commit                               4.3.0
prettytable                              2.5.0
priority                                 2.0.0
prometheus_client                        0.23.1
prometheus-fastapi-instrumentator        7.1.0
prompt_toolkit                           3.0.52
propcache                                0.3.2
protobuf                                 4.25.8
psutil                                   7.0.0
ptyprocess                               0.7.0
pure_eval                                0.2.3
py-cpuinfo                               9.0.0
py-spy                                   0.4.1
pyarrow                                  21.0.0
pyasn1                                   0.4.8
pyasn1-modules                           0.2.1
pybind11                                 3.0.0
pycodestyle                              2.8.0
pycountry                                24.6.1
pycparser                                2.22
pycryptodomex                            3.23.0
pydantic                                 2.11.4
pydantic_core                            2.33.2
pydantic-extra-types                     2.10.5
pydantic-settings                        2.10.1
pyelftools                               0.32
pyflakes                                 2.4.0
Pygments                                 2.19.2
PyGObject                                3.42.1
PyHamcrest                               2.0.2
PyJWT                                    2.3.0
pylint                                   3.3.8
PyMySQL                                  1.1.1
PyNaCl                                   1.5.0
pyOpenSSL                                21.0.0
pyparsing                                2.4.7
pyperclip                                1.9.0
pyrsistent                               0.18.1
pyserial                                 3.5
pysuinfer                                1.8.0
pysuinfer-llm                            2.0.0
pytest                                   7.3.2
pytest-cov                               6.2.1
python-apt                               2.4.0+ubuntu3
python-dateutil                          2.9.0.post0
python-debian                            0.1.43+ubuntu1.1
python-dotenv                            1.1.1
python-json-logger                       4.0.0
python-linux-procfs                      0.6.3
python-magic                             0.4.24
python-multipart                         0.0.20
python-pptx                              1.0.2
pytz                                     2025.2
pyudev                                   0.22.0
PyYAML                                   6.0.2
pyzmq                                    27.1.0
qdrant-client                            1.14.3
Quart                                    0.20.0
rados                                    2.0.0
ray                                      2.48.0
rbd                                      2.0.0
realtimeprofiler                         0.2.0+br1xx
redis                                    6.2.0
referencing                              0.36.2
regex                                    2024.11.6
requests                                 2.28.1
rich                                     14.0.0
rich-rst                                 1.3.1
rich-toolkit                             0.14.8
rignore                                  0.6.2
rpds-py                                  0.26.0
s3transfer                               0.13.1
safetensors                              0.5.3
schedule                                 1.2.2
scikit-learn                             1.7.0
scipy                                    1.15.3
SecretStorage                            3.3.1
sentence-transformers                    4.1.0
sentencepiece                            0.2.1
sentry-sdk                               2.33.0
service-identity                         18.1.0
setproctitle                             1.3.6
setuptools                               80.9.0
setuptools-scm                           9.2.1
shellingham                              1.5.4
shtab                                    1.7.2
six                                      1.17.0
smmap                                    5.0.2
sniffio                                  1.3.1
sos                                      4.5.6
soupsieve                                2.7
SQLAlchemy                               2.0.41
SQLAlchemy-Utils                         0.42.0
sse-starlette                            2.4.1
ssh-import-id                            5.11
stack-data                               0.6.3
starlette                                0.41.3
suinfer                                  2.5.0
sympy                                    1.13.3
systemd-python                           234
tabulate                                 0.9.0
taskgroup                                0.2.2
tenacity                                 9.1.2
tensorboard                              2.20.0
tensorboard-data-server                  0.7.2
threadpoolctl                            3.6.0
tiktoken                                 0.11.0
tokenizers                               0.21.2
tomli                                    2.2.1
tomli_w                                  1.2.0
tomlkit                                  0.13.3
torch                                    2.7.0+cpu
torchaudio                               2.7.0+cpu
torchdata                                0.11.0
torchvision                              0.22.0+cpu
tqdm                                     4.67.1
traitlets                                5.14.3
transformers                             4.53.1
transformers-br                          0.2.0+br1xx
treelib                                  1.8.0
triton                                   3.4.0
Twisted                                  22.1.0
typeguard                                4.4.4
typer                                    0.16.0
typing_extensions                        4.14.1
typing-inspection                        0.4.1
tyro                                     0.9.31
tzdata                                   2025.2
ubuntu-advantage-tools                   8001
ubuntu-drivers-common                    0.0.0
ufw                                      0.36.1
ujson                                    5.10.0
unattended-upgrades                      0.1
urllib3                                  1.26.13
uv                                       0.9.4
uvicorn                                  0.35.0
uvloop                                   0.21.0
virtualenv                               20.34.0
vllm                                     0.9.0.1
volcengine-python-sdk                    4.0.6
wadllib                                  1.3.6
waitress                                 3.0.2
wandb                                    0.21.3
watchfiles                               1.1.0
wcwidth                                  0.2.5
websockets                               15.0.1
Werkzeug                                 3.1.3
wheel                                    0.45.1
wrapt                                    1.17.3
wsproto                                  1.2.0
xgrammar                                 0.1.18
xkit                                     0.0.0
xlrd                                     2.0.2
xlsxwriter                               3.2.5
xxhash                                   3.5.0
yarl                                     1.20.1
zipp                                     3.23.0
zope.interface                           5.4.0
zstandard                                0.24.0

### Willingness to Implement | 实现意愿

- [ ] I'm willing to implement this myself | 我愿意自己解决
- [x] I would like someone else to implement this | 我希望其他人来解决

**Comment by angarchie** [comment]
Hey @DemingCheng! Thanks for the contribution — already solved the issue, thanks for helping us to hunt the bug! 🐞

Btw if you enjoyed contributing and want to stay connected with the project, we run a Discord server for contributors, a chill space to discuss features, ask questions, or pick up new issues  👉🏻 [MemOS Discord Server (Contributor Exclusive)](https://discord.gg/hszrwPAay8)

Thanks again for shipping this, and happy MemOSing 🧠✨
